ivy/functional/frontends/jax/random.py

```diff
@@ -73,3 +74,26 @@ def dirichlet(key, alpha, shape=None, dtype="float32"):
 def poisson(key, lam, shape=None, dtype=None):
     seed = _get_seed(key)
     return ivy.poisson(lam, shape=shape, dtype=dtype, seed=seed)
+
+
+@to_ivy_arrays_and_back
+@handle_jax_dtype
+@with_unsupported_dtypes(
+    {
+        "0.3.14 and below": (
+            "float16",
+            "bfloat16",
+        )
+    },
+    "jax",
+)
+def gumbel(key, shape=(), dtype="float64"):
+    seed = _get_seed(key)
+    uniform_x = ivy.random_uniform(
+        low=sys.float_info.min,
```

Comment:
We think that `0.0` is better than using `sys.float_info.min` to get the correct `gumbel` results, can you change that?